services/submission/src/test/java/app/coronawarn/server/services/submission/SubmissionPayloadGenerator.java

```diff
@@ -0,0 +1,89 @@
+package app.coronawarn.server.services.submission;
+
+import app.coronawarn.server.common.protocols.external.exposurenotification.ReportType;
+import app.coronawarn.server.common.protocols.external.exposurenotification.TemporaryExposureKey;
+import app.coronawarn.server.common.protocols.internal.SubmissionPayload;
+
+import com.google.protobuf.ByteString;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class SubmissionPayloadGenerator {
+
+  private static final int numberOfKeys = 10;
+  private static final int transmissionRiskLevel = 6;
+  private static final int rollingPeriod = 144; // 24*60/10
+  private static final ReportType reportType = ReportType.CONFIRMED_CLINICAL_DIAGNOSIS;
+  private static final ByteString requestPadding = ByteString.copyFrom(new byte[100]);
+  private static final List<String> visitedCountries = List.of("DE", "FR");
+  private static final String originCountry = "DE";
+  private static final boolean consentToFederation = true;
+  private static final int daysSinceOnsetOfSymptoms = 0;
```

Comment:
By convention, constants shall be uppercase with underscores, as it is the case for path to the payload below

Reply:
done